docs/resources/Guild.md

```diff
@@ -107,23 +107,23 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 
 ###### Guild Features
 
-| Feature                | Description                                                                      |
-| ---------------------- | -------------------------------------------------------------------------------- |
-| INVITE_SPLASH          | guild has access to set an invite splash background                              |
-| VIP_REGIONS            | guild has access to set 384kbps bitrate in voice (previously VIP voice servers)  |
-| VANITY_URL             | guild has access to set a vanity URL                                             |
-| VERIFIED               | guild is verified                                                                |
-| PARTNERED              | guild is partnered                                                               |
-| PUBLIC                 | guild is public                                                                  |
-| COMMUNITY              | guild is lurkable, can enable welcome screen, and receives public server updates |
-| COMMERCE               | guild has access to use commerce features (i.e. create store channels)           |
-| NEWS                   | guild has access to create news channels                                         |
-| DISCOVERABLE           | guild is able to be discovered in the directory                                  |
-| FEATURABLE             | guild is able to be featured in the directory                                    |
-| ANIMATED_ICON          | guild has access to set an animated guild icon                                   |
-| BANNER                 | guild has access to set a guild banner image                                     |
-| PUBLIC_DISABLED        | guild cannot be public                                                           |
-| WELCOME_SCREEN_ENABLED | guild has enabled the welcome screen                                             |
+| Feature                | Description                                                                     |
+| ---------------------- | ------------------------------------------------------------------------------- |
+| INVITE_SPLASH          | guild has access to set an invite splash background                             |
+| VIP_REGIONS            | guild has access to set 384kbps bitrate in voice (previously VIP voice servers) |
+| VANITY_URL             | guild has access to set a vanity URL                                            |
+| VERIFIED               | guild is verified                                                               |
+| PARTNERED              | guild is partnered                                                              |
+| PUBLIC                 | guild is public                                                                 |
+| COMMUNITY              | guild can enable welcome screen and discovery, and receives community updates   |
```

Comment:
```suggestion
| COMMUNITY              | guild can enable welcome screen and discovery, and receive community updates   |
```

Reply:
I was thinking of that as in "the guild receives community updates" since it's required for community guilds